src/blockchain_db/blockchain_db.cpp

```diff
@@ -417,8 +417,9 @@ void BlockchainDB::show_stats()
   );
 }
 
-void BlockchainDB::fixup()
+void BlockchainDB::fixup(fixup_context context)
```

Comment:
This (and in the subclasses) should be a const reference.

Reply:
Fixed